chroot-script

```diff
@@ -480,6 +480,10 @@ EOF
     local rootfs_mount_options=""
 
     if [ -z "${FILESYSTEM}" ] ; then
+      ## Debugging.
+      ls -la /dev/disk/by-uuid/"${TARGET_UUID}"
+      blkid /dev/disk/by-uuid/"${TARGET_UUID}"
```

Comment:
no reason to leave this commit in this PR, nor?

Reply:
Ok. Removed after force push. Done.